src/sentry/grouping/strategies/configurations.py

```diff
@@ -202,6 +202,18 @@ def register_strategy_config(id, **kwargs):
 )
 
 
+register_strategy_config(
```

Comment:
It would be cool if we could roll this change only for projects that have events using platform nextjs. That way we would not incur the dual grouping cost.

Reply:
I guess adding the change into an existing config makes the performance concern moot, though I'm going to work on having sdk filter in the rules syntax for limit blast radius of these rules.

Reply:
@armenzg  PR for restricting fingerprinting rules only to specific SDKs: https://github.com/getsentry/sentry/pull/62908